src/components/Sidebar/SearchResultsContainer.test.js

```diff
@@ -13,4 +13,15 @@ describe('searchResultsSelector', () => {
     expect(searchResultsSelector(testState))
       .toEqual(testSearchResults);
   });
+  it('should select fetching results from state', () => {
```

Comment:
let's be a bit more descriptive here: how about "should select fetching status from state"? Another option can be just using the variable name: "should select isFetching from state"